src/main.rs

```diff
@@ -254,7 +254,7 @@ fn main() {
         let path_mapping_file = path_mapping_file.to_owned();
         let path_mapping = Arc::clone(&path_mapping);
 
-        thread::spawn(move || {
+        thread::Builder::new().name(String::from("Producer")).spawn(move || {
```

Comment:
This one is still marked as new ;)

Reply:
@LucianaMarques try `git pull upstream master --rebase` and then `git push origin master --force`

Reply:
Ok, will do it today, sorry for the delay.

Reply:
`git pull upstream master --rebase
remote: Enumerating objects: 63, done.
remote: Counting objects: 100% (63/63), done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 73 (delta 57), reused 60 (delta 57), pack-reused 10
Unpacking objects: 100% (73/73), done.
From https://github.com/mozilla/grcov
 * branch            master     -> FETCH_HEAD
 * [new branch]      master     -> upstream/master
First, rewinding head to replay your work on top of it...
Applying: naming threads as they are created
Using index info to reconstruct a base tree...
M	src/main.rs
.git/rebase-apply/patch:12: trailing whitespace.
            
warning: 1 line adds whitespace errors.
Falling back to patching base and 3-way merge...
Auto-merging src/main.rs
CONFLICT (content): Merge conflict in src/main.rs
error: Failed to merge in the changes.
Patch failed at 0001 naming threads as they are created
Use 'git am --show-current-patch' to see the failed patch

Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort"`

And then when I try "git push origin master --force" it says it's already up-to-date

Reply:
It seems like your repo is in a bad state, maybe try `git reset --hard upstream master` and then reapply your patch manually, then commit and `git push origin master --force`.

Reply:
Hi Marco, I'm sorry for not making this faster!

I'll certainly do it, sorry for the delay.

Reply:
No need to apologize, you are helping us ;)